data/murderrun/function/main.mcfunction

```diff
@@ -6,8 +6,18 @@
+		execute if entity @e[tag=!levitationTrap,type=item, nbt={OnGround: 1b, Item: {components: {"minecraft:lore": ['{"extra":[{"color":"white","italic":false,"text":"When stepped on, causes the Seeker to float."}],"text":""}'], "minecraft:custom_name": '{"extra":[{"bold":true,"color":"white","italic":false,"text":"Levitation Trap"}],"text":""}'}, id: "minecraft:elytra"}}] as @e[tag=!levitationTrap, type=item,nbt={OnGround: 1b, Item: {components: {"minecraft:lore": ['{"extra":[{"color":"white","italic":false,"text":"When stepped on, causes the Seeker to float."}],"text":""}'], "minecraft:custom_name": '{"extra":[{"bold":true,"color":"white","italic":false,"text":"Levitation Trap"}],"text":""}'}, id: "minecraft:elytra"}}] run function murderrun:traps/levitation_trap/set_levitation_trap
 		execute if entity @e[tag=levitationTrap] as @e[tag=levitationTrap] at @s run function murderrun:traps/levitation_trap/levitation_trap
 
+	# Blind Trap
+		# /give @p minecraft:fermented_spider_eye[custom_name='["",{"text":"Blind Trap","color":"gray","bold":true,"italic":false}]',lore=['["", {"text":"When stepped on, gives the Seeker blindness.","color":"white","italic":false}]']]
+		execute if entity @e[tag=!blindTrap,type=item, nbt={OnGround: 1b, Item: {components: {"minecraft:lore": ['{"extra":[{"color":"white","italic":false,"text":"When stepped on, gives the Seeker blindness."}],"text":""}'], "minecraft:custom_name": '{"extra":[{"bold":true,"color":"gray","italic":false,"text":"Blind Trap"}],"text":""}'}, id: "minecraft:fermented_spider_eye"}}] as @e[tag=!blindTrap, type=item,nbt={OnGround: 1b, Item: {components: {"minecraft:lore": ['{"extra":[{"color":"white","italic":false,"text":"When stepped on, gives the Seeker blindness."}],"text":""}'], "minecraft:custom_name": '{"extra":[{"bold":true,"color":"gray","italic":false,"text":"Blind Trap"}],"text":""}'}, id: "minecraft:fermented_spider_eye"}}] run function murderrun:traps/blind_trap/set_blind_trap
+		execute if entity @e[tag=blindTrap] as @e[tag=blindTrap] at @s run function murderrun:traps/blind_trap/blind_trap
+
 # Scoreboard Resets (end of file)
 	execute if entity @a[scores={sneakTime=1..}] run scoreboard players reset @a[scores={sneakTime=1..}] sneakTime
+
```

Comment:
Can you remove the extra lines here? preferably, I'd like either 1 or 0 extra lines here

Reply:
Should be good now